tests/acceptance/apps/vhost-new-test.js

```diff
@@ -287,10 +287,6 @@ test(`visit ${appVhostsNewUrl} and create vhost with existing certificates`, fun
 
   signInAndVisit(appVhostsNewUrl);
 
-  stubRequest('post', '/accounts/stubbed-stack/certificates', function() {
-    assert.ok(false, 'should not create a new certificate');
-  });
-
```

Comment:
I did have failures. I put a break point in all the stubbed requests and that one never got hit. The assert(5) added up to 6 with it, so I removed it, but wasn't sure why it would have caused the test to fail.